packages/backend/src/ee/services/ai/utils/validateSelectedFieldsExistence.test.ts

```diff
@@ -0,0 +1,293 @@
+import {
+    DimensionType,
+    FieldType,
+    MetricType,
+    SupportedDbtAdapter,
+    type AdditionalMetric,
+    type CustomMetricBase,
+    type Explore,
+    type TableCalcsSchema,
+    type TableCalculation,
+} from '@lightdash/common';
+
+import { validateSelectedFieldsExistence } from './validators';
```

Comment:
Something I’ve tried in the filterExploreByTags tests is using a subset of the Explore type - a minimal version of it that function needs - so I don’t have to write huge mocks. It also makes the mocks more meaningful and readable